app.rb

```diff
@@ -0,0 +1,19 @@
+require 'pg'
+require 'sinatra/base'
+require './lib/peep.rb'
+
+class Chitter < Sinatra::Base
+
+  get '/' do
+    @peeps = Peep.all
+    erb(:index)
+  end
+
+  post '/postpeep' do
```

Comment:
Something I have to improve as well, again 😄 We should make the routes as RESTful as possible, in this case the route would look a little like that '/peep/new'